.github/workflows/5_testintegration_compile-and-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       if: matrix.os == 'ubuntu-latest'
       run: |
         sudo apt-get update
-        sudo apt-get install -y wget gnupg lsb-release software-properties-common \
+        sudo apt-get install -y wget cmake gnupg lsb-release software-properties-common \
```

Comment:
Originally cmake wasn't a dependency? should we check enforce some specific version?

Reply:
in that case let's avoid adding it.